deepcell_toolbox/utils.py

```diff
@@ -58,20 +59,25 @@ def erode_edges(mask, erosion_width):
         ValueError: mask.ndim is not 2 or 3
     """
     if erosion_width:
-        new_mask = np.zeros(mask.shape)
         if mask.ndim == 2:
-            strel = disk(erosion_width)
+            new_mask = np.copy(mask)
+            for i in range(erosion_width):
```

Comment:
`_` is used when we don't actually care about the variable itself.

```suggestion
            for _ in range(erosion_width):
```